### ISSUE TITLE
Invalid handling of query params for listVideos

### ISSUE THREAD

**Issue by boringContributor** [dan-online/bunnycdn-stream#90]
Hello there, it is me again,

I encountered another bug and wanted to notify you.

When calling `listVideos` the provided options do not work as you need to pass them as query params and not within the body of the request. 

I just had a look at `listVideos` for now so I'm not sure if this is impactful somewhere else.

The LOC I refer to -> https://github.com/dan-online/bunnycdn-stream/blob/main/src/index.ts#L250

How it should be -> https://docs.bunny.net/reference/video_list

You should take advantage of `URLSearchParams` something like:

```
const options = {
    page: '1', <-- string
    itemsPerPage: '10', <-- string
    search: 'hello',
    collection: '1312' <-- should rather be collectionId as you search for id and not the name
}

const urlParams = new URLSearchParams(options)
```

✌🏼

**Comment by dan-online** [comment]
Hey @boringContributor thanks for catching this! You are right it should be passed in the query, in this case by setting option.params with axios to that object, I'll open a PR in a bit to fix that 👍

**Comment by dan-online** [comment]
Published in ``bunnycdn-stream@2.1.3``
